i3lock.c

```diff
@@ -806,10 +805,7 @@ int main(int argc, char *argv[]) {
                 fprintf(stderr, "DPMS support has been removed from i3lock. Please see the manpage i3lock(1).\n");
                 break;
             case 'I': {
-                int time = 0;
-                if (sscanf(optarg, "%d", &time) != 1 || time < 0)
-                    errx(EXIT_FAILURE, "invalid timeout, it must be a positive integer\n");
-                inactivity_timeout = time;
+                fprintf(stderr, "Inactivity timeout makes only sense with DPMS. Please see the manpage i3lock(1).\n");
```

Comment:
Please add “, which was removed” so that people know what’s up without having to consult the manpage.